### PR TITLE
Release 3.2.7

### PR DESCRIPTION
**Pull request by Goose-Of-War**
# Review:
- Event asset folder rename
- Minute README modifications
- Poll optimizations
- Pre-commit for no cred users

### CODE REVIEW

**Review by nishkalprakash — APPROVED** [review]
Everything seems ok.

File: templates/poll_results.njk [hunk 1]
```diff
@@ -57,6 +57,7 @@
 		.bar {
 			width: 5px;
 			height: 24px;
+			margin-left: 5px;
```
Comment (nishkalprakash) [inline templates/poll_results.njk:60]: image
This broke the results page and added huge bars

Comment (nishkalprakash) [reply templates/poll_results.njk:60]: image
Also needs padding at the bottom

**Review by nishkalprakash — CHANGES_REQUESTED** [review]

**Comment by nishkalprakash** [comment]
Ah,
you already merged.
:(

**Comment by Goose-Of-War** [comment]
Come on. You said it seems ok

File: templates/poll_results.njk (hunk 1 above)
Comment (Goose-Of-War) [reply templates/poll_results.njk:60]: Padding in the footer or the result content?

**Comment by nishkalprakash** [comment]
But you put commits after that.
:/

File: templates/poll_results.njk (hunk 1 above)
Comment (ItsAnkan) [reply templates/poll_results.njk:60]: > image This broke the results page and added huge bars

where is it broken ;-;

**Comment by Goose-Of-War** [comment]
Gaah asddjhgsiudh

**Comment by nishkalprakash** [comment]
How do I test on the current poll,

When running dev,
I get the poll Goos for mayor or something

**Comment by Goose-Of-War** [comment]
Anyway, since it's merged, the one in kgpmask is the same as dev rn

File: templates/poll_results.njk (hunk 1 above)
Comment (nishkalprakash) [reply templates/poll_results.njk:60]: > Padding in the footer or the result content?

Padding between the results and the footer?

Comment (nishkalprakash) [reply templates/poll_results.njk:60]: the white scroll bars in the side and at the bottom

**Comment by Goose-Of-War** [comment]
Let's continue in #170

**Pull request merged**